app/models/invoice_item.rb

```diff
@@ -3,4 +3,12 @@
 class InvoiceItem < ApplicationRecord
   belongs_to :invoice
   belongs_to :item
+
+  def self.find_invoice_id(item_id)
```

Comment:
These could likely be methods in Item model since Items know about their Invoices through InvoiceItems-- depends on context in which its used but it cleans things up when applied to controllers/views since typically you'd always be referencing or iterating over items directly.